packages/babel-generator/src/index.js

```diff
@@ -117,7 +123,7 @@ function findCommonStringDelimiter(code, tokens) {
   if (occurences.single > occurences.double) {
     return "single";
   } else {
-    return "double";
+    return DEFAULT_STRING_DELIMITER;
```

Comment:
This usage seems weird to me since while it is the default, we don't specifically want the default here, we want the opposite of `single` which just happens to be the default. Personally I'd prefer us to leave this as-is and just use the string value where we want it.